modules/monitoring-applying-custom-alertmanager-configuration.adoc

```diff
@@ -104,6 +104,7 @@ receivers:
 ----
 +
 With this configuration, alerts of `critical` severity that are fired by the `example-app` service are sent using the `team-frontend-page` receiver. Typically these types of alerts would be paged to an individual or a critical response team.
+Also, because the `repeat_interval` value is set to less than the `group_interval` value, notifications will be repeated at each group interval.
```

Comment:
this only applies for the Watchdog alert. I'm a bit torn on mentioning such level of details...

Reply:
+1, if these are user oriented docs, I think this is getting too much into specific implementation details.

Reply:
Thanks, @simonpasquier and @danielmellado - can you ptal at the comments in the Jira issue and add your thoughts about how best to deal with the issue as described by the support engineer? https://issues.redhat.com/browse/RHDEVDOCS-185

Reply:
I removed this line from the PR. Are you both good with having `repeat_interval` set to 2m in the sample?